vmm/src/interrupt.rs

```diff
@@ -353,8 +354,26 @@ pub mod kvm {
                 kvm_route.u.msi.data = cfg.data;
 
                 if vm.check_extension(hypervisor::Cap::MsiDevid) {
+                    // On AArch64, there is limitation on the range of the 'devid',
```

Comment:
The limitation I mentioned in here: https://github.com/torvalds/linux/blob/v5.14/arch/arm64/kvm/vgic/vgic-its.c#L917

Reply:
@rbradford @sboeuf What do you think about this part?

Reply:
I think it's a great solution!